framework/opengl/gluDrawUtil.cpp

```diff
@@ -415,6 +415,7 @@ static void setVertexAttribPointer (const glw::Functions& gl, const VertexAttrib
 	if (isIntType && va.convert == VTX_COMP_CONVERT_NONE)
 		gl.vertexAttribIPointer(va.location, va.numComponents, compTypeGL, va.stride, va.pointer);
 	else
+		// TODO(kainino@chromium.org): Client side arrays are not allowed in WebGL. Emulate in Emscripten?
```

Comment:
You can try linking with `-s FULL_ES2=1`, that adds emulated support for rendering from client side arrays.